qa/tasks/cephfs/xfstests_dev.py

```diff
@@ -206,29 +205,44 @@ def install_deps(self):
                          'gettext', 'libedit-devel', 'libattr-devel',
                          'device-mapper-devel', 'libicu-devel']
 
-            deps_old_distros = ['xfsprogs-qa-devel']
-
-            if distro != 'fedora' and major_ver_num > 7:
-                    deps.remove('btrfs-progs-devel')
-
-            args = ['sudo', 'yum', 'install', '-y'] + deps + deps_old_distros
+            args = ['sudo', 'yum', 'install', '-y'] + deps
+        elif distro == 'fedora':
+            deps = """\
+                   acl attr automake bc dbench dump e2fsprogs fio gawk gcc \
+                   gdbm-devel git indent kernel-devel libacl-devel \
+                   libaio-devel libcap-devel libtool liburing-devel \
+                   libuuid-devel lvm2 make psmisc python3 quota sed sqlite \
+                   udftools xfsprogs \
+                   \
+                   btrfs-progs exfatprogs f2fs-tools ocfs2-tools xfsdump \
+                   xfsprogs-devel""".split()
+
+            args = ['sudo', 'yum', 'install', '-y'] + deps
         elif distro == 'ubuntu':
-            deps = """xfslibs-dev uuid-dev libtool-bin \
-            e2fsprogs automake gcc libuuid1 quota attr libattr1-dev make \
-            libacl1-dev libaio-dev xfsprogs libgdbm-dev gawk fio dbench \
-            uuid-runtime python sqlite3""".split()
+            k_rel = self.mount_a.client_remote.run(args='uname -r',
+                stdout=StringIO()).stdout.getvalue().strip()
+
+            deps = """\
+                   acl attr automake bc dbench dump e2fsprogs fio gawk \
+                   gcc git indent libacl1-dev libaio-dev libcap-dev \
+                   libgdbm-dev libtool libtool-bin liburing-dev libuuid1 \
+                   lvm2 make psmisc python3 quota sed uuid-dev uuid-runtime \
+                   xfsprogs sqlite3 \
+                   \
+                   exfatprogs f2fs-tools ocfs2-tools udftools xfsdump \
+                   xfslibs-dev""".split() + [f'linux-headers-{k_rel}']
```

Comment:
Here you're potentially asking to install a `linux-headers` package that only exists in chacra repos, but I don't think you've added the repo to the remote. If this is run with a non-distro kernel, this will likely fail.

Reply:
Got it.

Reply:
What about rest of 2 packages? `liburing-dev` and `exfatprogs`?